python/uv/__main__.py

```diff
@@ -21,37 +22,11 @@ def detect_virtualenv() -> str:
 
     return ""
 
-
-def find_uv_bin() -> str:
-    """Return the uv binary path."""
-
-    uv_exe = "uv" + sysconfig.get_config_var("EXE")
-
-    path = os.path.join(sysconfig.get_path("scripts"), uv_exe)
-    if os.path.isfile(path):
-        return path
-
-    if sys.version_info >= (3, 10):
-        user_scheme = sysconfig.get_preferred_scheme("user")
-    elif os.name == "nt":
-        user_scheme = "nt_user"
-    elif sys.platform == "darwin" and sys._framework:
-        user_scheme = "osx_framework_user"
-    else:
-        user_scheme = "posix_user"
-
-    path = os.path.join(sysconfig.get_path("scripts", scheme=user_scheme), uv_exe)
-    if os.path.isfile(path):
-        return path
-
-    raise FileNotFoundError(path)
-
-
-if __name__ == "__main__":
+def _run() -> None:
```

Comment:
By moving such logic into a private `_run` method, we avoid exposing all global variables as publicly import-able by anyone.